### PR TITLE
Bo's reactor deployment schedule

### PR DESCRIPTION
**Pull request by deniadjokic**
Hey Anthony,
I think you want this in your repo.
Denia

### CODE REVIEW

File: eg01-eg23.json [hunk 1]
```diff
@@ -0,0 +1,656 @@
...
+        "WASTE"
+       ]
+      }, 
+      "in_commod": {
+       "val": "FR Spent Fuel"
+      }, 
+      "input_capacity": "2000", 
+      "max_inv_size": "1.000000000000000E+299", 
+      "output_capacity": "2000", 
+      "repro_input_path": "hist/FR_reprocess.txt"
```
Comment (scopatz) [inline eg01-eg23.json:336]: This looks funny.

Comment (deniadjokic) [inline eg01-eg23.json:336]: I think you're not paying attention.

File: eg01-eg23.json [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+       }
+      ]
+     }, 
+     "name": "utility"
+    }, 
+    {
+     "config": {
+      "DeployInst": {
+       "buildorder": [
+        "<prototype>LWR</prototype><number>1</number><date>6</date>",
```
Comment (katyhuff) [inline eg01-eg23.json:441]: Interested in why these dates are in year rather than month. Is that handled elsewhere through the json sim def stuff?

Comment (scopatz) [inline eg01-eg23.json:441]: Nope this was a bug 12 x year that had since been fixed, though an updated json didn't get committed ;)

**Pull request merged**